config/crd/bases/pipelines.openshift.io_gitopsservices.yaml

```diff
@@ -35,6 +35,52 @@ spec:
             type: object
           spec:
             description: GitopsServiceSpec defines the desired state of GitopsService
+            properties:
+              infraNodeEnabled:
```

Comment:
If we agree to change the naming convention as suggested in the above comments. Please run `make manifests` and `make bundle` after the change.

Reply:
sure